pkg/registry/flowcontrol/ensurer/strategy.go

```diff
@@ -116,23 +116,27 @@ type configurationObject interface {
 // NewSuggestedEnsureStrategy returns an EnsureStrategy for suggested config objects
 func NewSuggestedEnsureStrategy() EnsureStrategy {
 	return &strategy{
-		alwaysAutoUpdateSpec: false,
-		name:                 "suggested",
+		alwaysAutoUpdateSpecFn: func(_ wantAndHave) bool {
+			return false
+		},
+		name: "suggested",
 	}
 }
 
 // NewMandatoryEnsureStrategy returns an EnsureStrategy for mandatory config objects
 func NewMandatoryEnsureStrategy() EnsureStrategy {
 	return &strategy{
-		alwaysAutoUpdateSpec: true,
-		name:                 "mandatory",
+		alwaysAutoUpdateSpecFn: func(_ wantAndHave) bool {
+			return true
+		},
```

Comment:
it will be replaced by
```
               func(wh wantAndHave) bool {
			if wh.getHave().GetName() == flowcontrolv1beta3.PriorityLevelConfigurationNameExempt {
				return false
			}
			return true
		},
```
in https://github.com/kubernetes/kubernetes/pull/117703

Reply:
This should probably test the object type too, we do not want this exception for a FlowSchema.